app/screens/send-bitcoin-screen/use-send-payment.ts

```diff
@@ -68,6 +68,13 @@ gql`
         message
       }
       status
+      transaction {
+        settlementVia {
+          ... on SettlementViaLn {
```

Comment:
please add
```
... on SettlementViaIntraLedger {
          preImage
        }
```
this will fix ln txs settled intraledger (just in case a provider uses blink as backend)